lib/routes/token.js

```diff
@@ -83,7 +83,7 @@ const PAYLOAD_SCHEMA = Joi.object({
       is: GRANT_JWT,
       then: Joi.forbidden()
     }),
-
+  redirect_uri: Joi.string().optional(),
```

Comment:
I suggest we do a little more validation here, e.g. checking that it doesn't contain any weirdy control characters etc.